[sharedmimeinfo] Unable to install

```
mv: cannot stat '.deps/update_mime_database-update-mime-database.Tpo': No such file or directory
make[1]: *** [Makefile:612: update_mime_database-update-mime-database.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/usr/local/tmp/crew/shared-mime-info-1.9.tar.xz.dir/shared-mime-info-1.9'
make: *** [Makefile:716: check-recursive] Error 1
sharedmimeinfo failed to build: `make V=0 -j4` exited with 2
```

--- a/packages/sharedmimeinfo.rb
+++ b/packages/sharedmimeinfo.rb
@@ -25,7 +25,7 @@ class Sharedmimeinfo < Package
 
   def self.build
     system "./configure --prefix=#{CREW_PREFIX} --libdir=#{CREW_LIB_PREFIX}"
-    system "make"
+    system "make -j1"
   end
 
   def self.install

--- a/packages/sharedmimeinfo.rb
+++ b/packages/sharedmimeinfo.rb
@@ -8,10 +8,14 @@ class Sharedmimeinfo < Package
   source_sha256 '5c0133ec4e228e41bdf52f726d271a2d821499c2ab97afd3aa3d6cf43efcdc83'
 
   binary_url ({
+    aarch64: 'https://dl.bintray.com/chromebrew/chromebrew/sharedmimeinfo-1.9-chromeos-armv7l.tar.xz',
+     armv7l: 'https://dl.bintray.com/chromebrew/chromebrew/sharedmimeinfo-1.9-chromeos-armv7l.tar.xz',    
        i686: 'https://dl.bintray.com/chromebrew/chromebrew/sharedmimeinfo-1.9-chromeos-i686.tar.xz',
      x86_64: 'https://dl.bintray.com/chromebrew/chromebrew/sharedmimeinfo-1.9-chromeos-x86_64.tar.xz',
   })
   binary_sha256 ({
+    aarch64: '360a392b05c880870a927cfa6ea947c831a8d4de96a6a082bd03b9b419e97813',
+     armv7l: '360a392b05c880870a927cfa6ea947c831a8d4de96a6a082bd03b9b419e97813',    
        i686: 'a3c19ef00a5d2623156999ed2bb594d9a3c5ca01c15917081ca40bdccad07cf8',
      x86_64: '95d620b1c585e98299d12e5de9a160a261a4acfdab135563c16f52112bd05d10',
   })

